visualize-admin/visualization-tool
# fix: Fixed tooltip for Mobile Area Charts

@bprusinowski please check if this is the correct approach, if so I will continue implementing this for all charts and for tooltips that are larger the arrow will have to be more dynamic of course.

This PR: 
- Fixes Mobile Tooltip for all Chart except (Pie Charts, Scatterplot Charts, Map Charts)
- The exceptions still need design criteria, @KerstinFaye 
- This PR also uncovered a new Issue [#110](https://github.com/visualize-admin/visualization-tool-private/issues/110)
- This PR is the start to fix Issue [#94](https://github.com/visualize-admin/visualization-tool-private/issues/94)

This PR will resolve #94

---
### How to Test
1. Go to this [Visit Preview](https://vercel.live/open-feedback/visualization-tool-git-fix-mobile-tooltip-positioning-ixt1.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)
2. Use the Einmalvergütung für Photovoltaikanlagen Dataset (you can test almost all Chart types w/ this)

3. Select Bar Chart -> Publish -> See how the tooltip is at the bottom with adjusting arrow ✅
4. Select Line Chart -> Publish -> See how the tooltip is at the bottom with adjusting arrow ✅
5. Select Area Chart -> Publish -> See how the tooltip is at the bottom with adjusting arrow ✅
6. Select Scatter Chart -> Publish -> See how the tooltip is at the bottom with adjusting arrow, but not optimal design ✅
7. Select Pie Chart -> Publish -> See how the tooltip is not placed correctly on mobile (potential separate issue), but is highlighted correctly ✅
8. Select Dual Axis Line Chart -> Publish -> See how the tooltip is at the bottom with adjusting arrow, but not optimal design ✅
9. Select Column Line Chart -> Publish -> See how the tooltip is at the bottom with adjusting arrow ✅

---

CC: @KerstinFaye , @sosiology
